lib/CLI/Parser.hs

```diff
@@ -443,6 +449,9 @@ expectedFilterFormat = Pretty.vsep
 -- Utils
 ----------------------------------------------------------------------------
 
+getBackendsHelpMsg :: (SupportedBackend -> Pretty.Doc) -> [Pretty.Doc]
```

Comment:
`getBackendsHelpMsg`'s first argument is always `bPathHelpMsg`, so it looks like it doesn't really need the `SupportedBackend -> Pretty.Doc` parameter.

Reply:
I wanted to keep space for other help msgs
But, ok, will get rid of it

Reply:
done
So it was too small to be a function, so I left a pure map